monkestation/code/modules/reagents/chemistry/machinery/atmos_machine.dm

```diff
@@ -0,0 +1,172 @@
+#define REAGENTS_BASE_VOLUME 100 // actual volume is REAGENTS_BASE_VOLUME plus REAGENTS_BASE_VOLUME * rating for each matterbin
+
+///dont tell anyone this but this is literally just smoke machine code but with names replaced and smoke replaced with gas -Borbop
```

Comment:
Why not just replace the smoke machine?
It doesn't make too much sense to have "smoke" and "atmospherics" as separate things

Reply:
because smoke still exists i was thinking of replacing it, but smoke has different "functions" from the atmos machine